IFC4x3/Properties/i/InflowSum_2OMwvg2WD9yQR5E1WntDRu/Documentation.md

```diff
@@ -0,0 +1 @@
+Groundwater inflow measured or expected durng a certain time span.
```

Comment:
The time span part confuses me. Is it supposed to be `IfcVolumeMeasure`? Otherwise, I would expect an absolute value such as "Groundwater inflow measured or expected durng a one hour."

Reply:
Agreed! Ping @j-weil.

Reply:
Updated description.